internal/app/mapper/mock/mapper.go

```diff
@@ -0,0 +1,11 @@
+package mapper
+
+import (
+	"testing"
+
+	"github.com/envoyproxy/xds-relay/internal/app/mapper"
+)
+
+func NewMapper(t *testing.T) mapper.Mapper {
```

Comment:
do you think it'd be cleaner to move this to `internal/app/mapper/mapper_mock.go`? (Same with `/mock/orchestrator.go`)

Reply:
I moved all functions from the mock subdirectories (for mapper, orchestartor, and upstream client) to the `*_mock.go` files and deleted the subdirectories. Had to rename some functions but hopefully the structure's less confusing now